pkgs/by-name/ad/addr-book-combine/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromSourcehut,
+  ...
+}:
+buildGoModule {
+  pname = "addr-book-combine";
+  version = "2022-12-15";
+
+  src = fetchFromSourcehut {
+    owner = "~jcc";
+    repo = "addr-book-combine";
+    rev = "11696f4726b981c774ad1d4858f1a935ac71e9ac";
+    hash = "sha256-SENur3p5LxMNnjo/+qiVdrEs+i+rI1PT1wYYdLLqWrg=";
+  };
+
+  vendorHash = null;
```

Comment:
may need an actual hash

Reply:
It doesn't since it doesn't have any dependencies.